docs/src/main/sphinx/client/client-protocol.md

```diff
@@ -17,6 +17,9 @@ The protocol is a sequence of REST API calls to the
 6. The client and coordinator continue with steps 4. and 5. until all
    result set data is returned to the client or the client stops requesting
    more data.
+7. If the client fails to fetch the result set, the coordinator does not initiate
+   further processing, fails the query, and returns a `USER_CANCELED` error.
```

Comment:
technically speaking query is processing as long as the output buffers are not pooled and the query is not marked as abandoned after a configurable client timeout has passed
@mosabua

Reply:
That might be an important detail for the dev guide but I think for the overview in the user guide that is too much detail.